gwt-material/src/main/java/gwt/material/design/client/base/mixin/InitialClassMixin.java

```diff
@@ -0,0 +1,64 @@
+package gwt.material.design.client.base.mixin;
+
+/*
+ * #%L
+ * GwtMaterial
+ * %%
+ * Copyright (C) 2015 - 2016 GwtMaterialDesign
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+
+import com.google.gwt.event.logical.shared.AttachEvent;
+import com.google.gwt.user.client.ui.UIObject;
+import com.google.gwt.user.client.ui.Widget;
+import gwt.material.design.client.base.HasInitialClass;
+
+/**
+ * @author kevzlou7979
+ */
+public class InitialClassMixin<T extends UIObject & HasInitialClass> extends AbstractMixin<T> implements HasInitialClass {
+
+    private String initialClass;
+
+    public InitialClassMixin(final T uiObject) {
+        super(uiObject);
+    }
+
+
+    @Override
+    public void setInitialClass(final String initialClass) {
+        this.initialClass = initialClass;
```

Comment:
Every time `setInitialClass` is called it adds a new attach handler without removing the existing one. Causing unexpected behavior when called multiple times. Should change this to remove the existing handler before creating a new one.